dashboards/sap-hana-details.json

```diff
@@ -930,36 +964,55 @@
       ],
       "targets": [
         {
-          "expr": "hanadb_disk_used_size_mb{host=~\"$node_name\",sid=~\"$sid\",insnr=~\"$instance_number\",database_name=~\"$database_name\"} + 0",
+          "expr": "hanadb_disk_used_size_mb{host=~\"$node_name\",sid=~\"$sid\",insnr=~\"$instance_number\",database_name=~\"$database_name\",usage_type!=\"ROOTKEY_BACKUP\"} + 0",
           "format": "table",
           "instant": true,
           "interval": "",
           "intervalFactor": 1,
-          "legendFormat": "Used Disk",
+          "legendFormat": "Usage by type",
```

Comment:
Should this not be the `Used Disk`/`Usage`

Reply:
I changed this legend because this was not actually the usage of the disk, just how much of the disk that specific _type_ was consuming.